gateway/1.4/modules/ROOT/pages/flex-gateway-k8-high-availability.adoc

```diff
@@ -0,0 +1,269 @@
+--reuse-values \
+--set replicaCount=<number of replicas>
+----
++
+.Example:
+[source,helm]
+----
+helm -n gateway upgrade \
+ingress flex-gateway/flex-gateway \
+--reuse-values \
+--set replicaCount=2
+----
++
+When successful, the command returns output similar to this: 
++
+----
+Release "ingress" has been upgraded. Happy Helming!
+NAME: ingress
+LAST DEPLOYED: Mon Apr 17 15:00:09 2023
+NAMESPACE: gateway
+STATUS: deployed
+REVISION: 27
+TEST SUITE: None
+----
+
+. If the Helm chart is not installed, run the following command:
++
+.Syntax:
+----
+helm -n <namespace> \
+upgrade -i --create-namespace \
+<release-name> <helm-repository-name>/<helm-chart-name> \
+--set-file registration.content=<registration file> \
+--set replicaCount=<number of replicas>
+----
++
+.Example:
+[source,helm]
+----
+helm -n gateway \
+upgrade -i --create-namespace \
+ingress flex-gateway/flex-gateway \
+--set-file registration.content=registration.yaml \
+--set replicaCount=2
+----
++
+When successful, the command prints output similar to this example: 
++
+----
+Release "ingress" does not exist. Installing it now.
+NAME: ingress2
+LAST DEPLOYED: Mon Apr 17 15:32:50 2023
+NAMESPACE: gateway
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+----
+
+. Verify creation of the replicas by running the following command:
++
+.Syntax:
+----
+kubectl get rs -n <namespace>
+----
++
+.Example:
+----
+kubectl get rs -n gateway
+----
++
+When successful, the command prints output similar to this example: 
++
+----
+NAME                  DESIRED   CURRENT   READY   AGE
+ingress-5b7474b8f6    2         2         2       70s
+----
+
+[[autoscale]]
+== Configure Autoscaling
+
+Horizontal Pod autoscaling generates Flex Gateway replicas automatically based on CPU, memory, and other target and behavioral settings. 
+
+=== Autoscaling Parameters
+
+To use autoscaling, enable Horizontal Pod Autoscaler (HPA), and configure other Helm chart settings to define autoscaling . Autoscaling parameters in the Helm chart follow.
+
+[%header,cols="1,2a"]
+|===
+| Parameter | Description
+| `autoscaling.enabled` | Boolean that indicates whether the Horizontal Pod Autoscaler (HPA) is enabled. Defaults to `false`.
+| `autoscaling.minReplicas` | The minimum number of replicas that the HPA scaler is allowed to create. Defaults to `2`.
+| `autoscaling.maxReplicas` | The maximum number of replicas that the HPA scaler is allowed to create. Defaults to `11`.
+| `autoscaling.targetCPUUtilizationPercentage` | A resource metric that sets the average CPU usage percentage of all deployed Pods. Defaults to `50`.
+| `autoscaling.targetMemoryUtilizationPercentage` | A string representing the average memory usage percentage of all deployed Pods. Defaults to `nil`.
+| `autoscaling.extraMetrics` | A list of custom Pod configuration metrics that enables you to collect and monitor application-specific metric data. Defaults to `[]`. Define the metrics with nested parameters, for example:
```

Comment:
from Israel, get data with commands like this:
kubectl get --raw /apis/metrics.k8s.io/v1beta1/nodes/<NODE_NAME>
kubectl get --raw /apis/metrics.k8s.io/v1beta1/namespaces/<NAMESPACE>/pods/<POD_NAME>